solidity/test/e2e/Common.sol

```diff
@@ -106,22 +119,34 @@ contract CommonE2EBase is DSTestPlus, TestConstants {
     );
 
     /// =============== NON HOME CHAIN ===============
+    vm.selectFork(_OPTIMISM_FORK_ID);
```

Comment:
For the proofs we will need to do more then forking, we need to broadcast them as we want the contracts acctually on the anvil nodes to get the proofs